### PR TITLE
changed percent_proliferating_cells, percent_inflammatory_tissue, per…

### PR DESCRIPTION
**Pull request by hknahal**
…cent_stromal_cells and percent_necrosis fields from required to optional after confirmation from TCA WG

### CODE REVIEW

**Comment by hknahal** [comment]
Related to https://github.com/icgc-argo/argo-dictionary/issues/214

**Comment by rosibaj** [comment]
i have completed the work to remove hte mandatory, and updated the descriptions per my review as well.

**Review by rosibaj — APPROVED** [review]

File: schemas/specimen.json [hunk 1]
```diff
@@ -346,7 +344,6 @@
       "description": "Indicate a value, in decimals, that represents the percentage of reactive cells that are present in a malignant tumour specimen but are not malignant such as fibroblasts, vascular structures, etc.",
       "valueType": "number",
       "meta": {
-        "core": true,
         "dependsOn": "sample_registration.tumour_normal_designation",
         "notes": "#/notes/if_tumour",
```
[the remark this answers is not in the document]
Comment (hknahal) [reply schemas/specimen.json]: @rosibaj Yes, I agree, the note sounds like they are still required so better to remove it. We mention "tumour specimen" in the field description so it's clear the field is only applicable to tumour specimens.

File: schemas/specimen.json [hunk 2]
```diff
@@ -335,7 +334,6 @@
       "description": "Indicate a value, in decimals, that represents local response to cellular injury, marked by capillary dilatation, edema and leukocyte infiltration; clinically, inflammation is manifest by redness, heat, pain, swelling and loss of function, with the need to heal damaged tissue.",
       "valueType": "number",
       "meta": {
-        "core": true,
         "dependsOn": "sample_registration.tumour_normal_designation",
```
[the remark this answers is not in the document]
Comment (hknahal) [reply schemas/specimen.json:337]: Yes @rosibaj

**Pull request merged**